### ISSUE TITLE
CTAN: update support link of CTeX bbs on CTAN

### ISSUE THREAD

**Issue by Liam0205** [CTeX-org/ctex-kit#393]
Currently, <http://bbs.ctex.org/forum.php> is set for several packages on CTAN. However, the forum was shut down since Nov. 21, 2018.

![image](https://user-images.githubusercontent.com/3348485/49282095-38011900-f4c9-11e8-879e-e5ddcbfc712e.png)

Early today, an email from the CTAN team shows concern of the dead link problem. Since @wulingyun hasn't posted any response to [this thread](https://groups.google.com/forum/#!topic/ctex/kKb50zsl4fU), we don't have any valuable response about this issue right now. In the response to the team, the situation was addressed, and the team said they will not remove the link until any new update is synced to them.

This issue is opened for a tracking, and for a discussion about replacing <http://bbs.ctex.org/forum.php> to <https://github.com/CTeX-org/ctex-kit/issues/>, if the bbs will never be rebooted again.

So, let's vote for the new link.

**Comment by Liam0205** [comment]
@Liam0205 plus one for <https://github.com/CTeX-org/ctex-kit/issues/>.

**Comment by Liam0205** [comment]
The original email from the team.

> Hi folks,
> The link http://bbs.ctex.org/forum.php which appears in catalogue
> entries of packages

> ctex-faq, ctex, zhnumber, zhlineskip, xecjk and xpinyin

> is dead.

> Please advise. Thank you.

**Comment by wulingyun** [comment]
Please replace the link to GitHub.

I have replied to google groups in Nov. 27, and it seems there was something wrong with the reply email.

The forum was closed due to the administrative requirements of beian (see www.beian.gov.cn for more information). Since it is difficult to meet the requirements, I can not see any chance to reopen the forum in near future.

**Comment by Liam0205** [comment]
@wulingyun Got it. 

Finally we have an official response.

Appreciation should be shown to you, Linyun, for your continuous support for the CTeX site and forum, and our community. 

Best!

**Comment by wulingyun** [comment]
@Liam0205 My pleasure.

If you guys have any suggestion on the substitution of forum or the website, please let me know.

Thanks all guys for great efforts on ctex-kit and other projects.

**Comment by wulingyun** [comment]
@Liam0205 Agree, it is also what I want to do. But I did not find any existing approach to build the archive from Discuz. I will try to find some solution.

**Comment by Liam0205** [comment]
@wulingyun 
After searching on Google, I found this old thread that might be helpful for the archiving task. Please check it when you're available.

* <http://www.discuz.net/thread-61742-1-1.html>

**Comment by wulingyun** [comment]
@Liam0205 Thanks.

It is for the very old Discuz 3.12. The current version is Discuz X3.3, which is quite different from the interface to database.

**Comment by Liam0205** [comment]
@wulingyun Well, I didn't notice it. So we'll have to find some alternatives.

**Comment by Liam0205** [comment]
@wulingyun If the database of the forum is available to read, and no official archive method could be found in near future, parsing the database and build a stataic site for search engine indexing is valuable for corsideration.
